pkg/operator/ceph/csi/provisioner/controller.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2019 The Rook Authors. All rights reserved.
```

Comment:
this file is mostly adapted from https://github.com/kubernetes-csi/external-provisioner/blob/master/cmd/csi-provisioner/csi-provisioner.go

Should we also pick up Kubernetes Authors in copyright?

Reply:
Yes, sounds good to add a line giving the copyright